.azure-pipelines/client.yml

```diff
@@ -27,39 +30,40 @@ jobs:
         inputs:
           ArtifactName: packages
 
-  - job: 'Test_Windows'
+  - job: 'Analyze'
+
+    dependsOn:
+      'Build_Packages'
 
     pool:
       vmImage: 'vs2017-win2016'
 
     steps:
-      - task: DotNetCoreInstaller@0
-        displayName: 'Use .NET Core sdk 2.1.503'
-        inputs:
-          version: 2.1.503
-
-      - task: DotNetCoreCLI@2
-        displayName: 'Build & Test'
-        inputs:
-          command: test
-          projects: '$(ProjectFile)'
-
       - task: ms.vss-governance-buildtask.governance-build-task-component-detection.ComponentGovernanceComponentDetection@0
         displayName: 'Component Detection'
         # ComponentGovernance is currently unable to run on pull requests of public projects.  Running on
         # scheduled builds should be good enough.
-        condition: and(succeeded(), eq(variables['Build.Reason'], 'Schedule'))
+        condition: and(succeededOrFailed(), ne(variables['Build.Reason'],'PullRequest'))
+
+  - job: 'Test'
 
-  - job: 'Test_Linux'
+    strategy:
+      matrix:
+        Linux:
+          os.name: 'Linux'
+          os.vmImage: 'ubuntu-16.04'
```

Comment:
Rename to `VmImage` to match new style guideline.

Reply:
Actually we've been using similar namings in strategy matrix, in all other repos
https://github.com/Azure/azure-sdk-for-js/blob/master/.azure-pipelines/client.yml
Will we need to update others too then?

Reply:
I'll change it to OSVmImage, for consistency with other repos. Because we'll make similar changesin other repos

Reply:
I prefer `VmImage` to `OSVmImage`, but I'm fine with either to prevent blocking this PR for longer.  In addition to standardizing yaml style, we'll also want to standardize names for common variables.